nixos/doc/manual/redirects.json

```diff
@@ -524,9 +524,6 @@
   "module-services-nextcloud-known-warnings": [
     "index.html#module-services-nextcloud-known-warnings"
   ],
-  "module-services-nextcloud-warning-logreader": [
-    "index.html#module-services-nextcloud-warning-logreader"
-  ],
```

Comment:
Could you instead prepend an entry here that points to a section in the release notes stating why this was removed?
cc @fricklerhandwerk

Reply:
I can try, but I'm not sure how to properly do that, also given the mentioned PR which seems to restructure the changelogs.

Reply:
What should we write there? "It is no longer relevant". I'm having my fair issue thinking this would help anyone.

Reply:
We could note that the log reader application was fixed, or whatever the reason is for this note to get removed.